www/docs/en/dev/config_ref/images.md

```diff
@@ -42,79 +45,107 @@ density       | *Optional* <br/> Specified icon density (Android Specific)
 
 The following configuration can be used to define single default icon
 which will be used for all platforms.
-
-        <icon src="res/icon.png" />
-
+```xml
+    <icon src="res/icon.png" />
+```
 For each platform you can also define a pixel-perfect icons set to fit
 different screen resolutions.
 
 ##Android
-
-         <platform name="android">
-                  <icon src="res/android/ldpi.png" density="ldpi" />
-                  <icon src="res/android/mdpi.png" density="mdpi" />
-                  <icon src="res/android/hdpi.png" density="hdpi" />
-                  <icon src="res/android/xhdpi.png" density="xhdpi" />
-         </platform>
+```xml
+    <platform name="android">
+        <!-- 
+            ldpi    : 36x36 px
+            mdpi    : 48x48 px
+            hdpi    : 72x72 px
+            xhdpi   : 96x96 px
+            xxhdpi  : 144x144 px
+            xxxhdpi : 192x192 px
+        -->
+        <icon src="res/android/ldpi.png" density="ldpi" />
+        <icon src="res/android/mdpi.png" density="mdpi" />
+        <icon src="res/android/hdpi.png" density="hdpi" />
+        <icon src="res/android/xhdpi.png" density="xhdpi" />
+        <icon src="res/android/xxhdpi.png" density="xxhdpi" />
+        <icon src="res/android/xxxhdpi.png" density="xxxhdpi" />
+    </platform>
+```
+###See Also
+- [Android icon guide](https://www.google.com/design/spec/style/icons.html)
+- [Android - Supporting multiple screens](http://developer.android.com/guide/practices/screens_support.html)
```

Comment:
It's easier to see them all in the same place at the bottom, but it doesn't make much of a difference.